closure_linter/javascriptlintrules.py

```diff
@@ -287,9 +287,9 @@ def CheckToken(self, token, state):
           elif (not function.has_return and
                 not function.has_throw and
                 function.doc and
-                function.doc.HasFlag('return') and
+                (function.doc.HasFlag('returns') or function.doc.HasFlag('return')) and
                 not state.InInterfaceMethod()):
-            flag = function.doc.GetFlag('return')
+            flag = function.doc.GetFlag('returns') if function.doc.GetFlag('returns') else function.doc.GetFlag('return')
```

Comment:
You can write this more idiomatic as:
flag = function.doc.GetFlag('return') or function.doc.GetFlag('returns')